docs/content/docs/install/settings.md

```diff
@@ -250,6 +250,16 @@ A few settings are available to configure this feature:
 
   Set this to the root URL of your custom console. example: `https://mycorp.com`
 
+* `custom-console-url-namespace`
```

Comment:
For openshift we don't need to set this right ?

asking because when i test this changes i don't see namespace i details

![Screenshot from 2023-11-06 16-48-57](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/441a35bb-13d2-4b48-97f1-f7a86df454e6)

Reply:
I mean i tried manually i don't see Namespace and PR name in details :thinking: